content/recipes/accessibility/sftp.md

```diff
@@ -254,17 +254,17 @@ For example [Rclone](https://rclone.org) is one such tool, allowing interaction
 
 ### What to read next?
 
-> [Wikipedia article on SFTP](https://en.wikipedia.org/wiki/SSH_File_Transfer_Protocol)
+> * [Wikipedia article on SFTP](https://en.wikipedia.org/wiki/SSH_File_Transfer_Protocol)
 >
-> [The Geek Stuff, FTP and SFTP Beginners guide with 10 examples](https://www.thegeekstuff.com/2010/06/ftp-sftp-tutorial/)
+> * [The Geek Stuff, FTP and SFTP Beginners guide with 10 examples](https://www.thegeekstuff.com/2010/06/ftp-sftp-tutorial/)
 >
-> [Example of customization for a specific institution: University of Luxembourg File Transfer](https://hpc.uni.lu/users/docs/filetransfer.html)
+> * [Example of customization for a specific institution: University of Luxembourg File Transfer](https://hpc.uni.lu/users/docs/filetransfer.html)
 >
-> [Setting up an SFTP server on CentOS](https://www.howtoforge.com/tutorial/how-to-setup-an-sftp-server-on-centos/)
+> * [Setting up an SFTP server on CentOS](https://www.howtoforge.com/tutorial/how-to-setup-an-sftp-server-on-centos/)
 >
-> {ref}`fcb-access-aspera`
+> * {ref}`fcb-access-aspera`
 >
-> [Rclone](https://rclone.org)
+> * [Rclone](https://rclone.org)
```

Comment:
[Rclone for managing files on cloud storage]((https://rclone.org)

Adding a short explanation about Rclone